homework.js

```diff
@@ -0,0 +1,88 @@
+// Q1:
+
+function sumNum(n) {
```

Comment:
it's a better practice to define a function using expression.
`const sumNum = function(n){}`